socorro/tests/external/es/test_supersearch.py

```diff
@@ -500,35 +494,46 @@ def test_get_with_combined_operators(self, es_helper):
         es_helper.index_crash(
             processed_crash={
                 "uuid": create_new_ooid(timestamp=now),
-                "signature": sigs[3],
+                "signature": "foo(bar)",
                 "app_notes": "mozilla bar",
                 "product": "EarthRacoon",
                 "date_processed": now,
             },
         )
         es_helper.refresh()
 
+        # Search for signatures matching "js" or containing "::"
         res = api.get(signature=["js", "~::"])
         assert res["total"] == 3
-        assert sorted([x["signature"] for x in res["hits"]]), sorted(
```

Comment:
This always passes because it mistakenly uses a `,` instead of a `==` between the two expressions. So the assertion is asserting that `sorted([x["signature"] for x in res["hits"]])` is truthy which it is unless it's an empty list. That's not what we want, though. We want to compare the first expression to the second with a `==`. I fixed that.